### DISCUSSION TITLE
hey

### DISCUSSION THREAD

**Discussion by dinithrusiru** [community/community#124409] [Code Security]
### Select Topic Area

Question

### Body

how increase securaty

**Comment by dinith-rusiru** [comment]
add more securaty methods

**Comment by ebndev** [comment]
Hi @dinithrusiru & @it22145284, thanks for being a part of the GitHub Community!

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
